sevntu-checkstyle-sonar-plugin/import-control.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<!DOCTYPE import-control PUBLIC
+    "-//Puppy Crawl//DTD Import Control 1.2//EN"
+    "http://checkstyle.sourceforge.net/dtds/import_control_1_2.dtd">
+
+<import-control pkg="com.github.sevntu.checkstyle">
+
+  <allow pkg="java"/>
+  <allow pkg="org"/>
```

Comment:
OK, will do. To be honest I didn't give this much thought - it's two just old classes.. ;-)